Application.mk

```diff
@@ -138,11 +138,11 @@ $(CXXOBJS): %$(SUFFIX)$(OBJEXT): %$(CXXEXT)
 	$(if $(and $(CONFIG_BUILD_LOADABLE),$(CXXELFFLAGS)), \
 		$(call ELFCOMPILEXX, $<, $@), $(call COMPILEXX, $<, $@))
 
-archive:
+archive: $(OBJS)
```

Comment:
should we chnage archive to a real target? then we can avoid the achive every time even there is nothing to change.